frontend/src/api/client/models/Message.ts

```diff
@@ -6,7 +6,7 @@ import type { FunctionCall } from './FunctionCall';
 import type { RegularMessage } from './RegularMessage';
 export type Message = {
```

Comment:
Why not two separate types? You would be able to avoid calling typeof later

Reply:
Checking type not from the schema but from property is easier since it is defined from the backend